.zenodo.json

```diff
@@ -0,0 +1,29 @@
+{
+    "description": "A flexible tree-based numerical simulation framework for PDEs written in Julia",
+    "license": "mit-license",
```

Comment:
Shouldn't this be just `MIT`? (see https://licenses.opendefinition.org/licenses/MIT.json) (but I am not an expert...)

Reply:
I'll change that. It looks like new stuff has happened there - we could only use `mit-license`, which is still included in `legacy_ids` there.